File: 프로그래머스/0/181875. 배열에서 문자열 대소문자 변환하기/배열에서 문자열 대소문자 변환하기.java
class Solution {
    public String[] solution(String[] strArr) {
        // 배열 선언 및 초기화 - 배열의 길이는 strArr과 같음
        String[] answer = new String[strArr.length];
        // 홀수번째와 짝수번째를 구분해 적용할 for문 - toUpperCase() & toLowerCase() 사용
        for(int i=0; i<strArr.length; i++){
            if(i%2 == 0){
                answer[i] = strArr[i].toLowerCase();
            } else {
                answer[i] = strArr[i].toUpperCase();
            }
        }
            
        // 정답 리턴
        return answer;
    }
}

File: 프로그래머스/0/181875. 배열에서 문자열 대소문자 변환하기/배열에서 문자열 대소문자 변환하기.py
def solution(strArr):
    answer = []
    i = 0
    
    while i < len(strArr):
        if i % 2 == 0:
            answer.append(strArr[i].lower())
            i += 1
        else:
            answer.append(strArr[i].upper())
            i += 1
            
    return answer